src/prerender.ts

```diff
@@ -37,12 +37,7 @@ export async function prerender(nitro: Nitro) {
     routes.add(route);
   }
 
-  // Crawl / at least if no routes are defined
-  if (nitro.options.prerender.crawlLinks && routes.size === 0) {
```

Comment:
Shall we move the check after hook in meanwhile and discuss about v3 behavior later? 🙏🏼

Reply:
looks great! thank you for updating it ❤️